chapters/fr/slides/chapter2_01_data-structures-1.md

```diff
@@ -64,25 +64,25 @@ devons toujours fournir le vocabulaire partagé.
 - Recherche de la chaine et du hash dans `nlp.vocab.strings`
 
 ```python
-doc = nlp("I love coffee")
-print("hash value:", nlp.vocab.strings["coffee"])
-print("string value:", nlp.vocab.strings[3197928453018144401])
+doc = nlp("Inès aime nager")
```

Comment:
```suggestion
doc = nlp("Ines aime nager")
```

I just don't want people to get confused and think my name actually has an accent like this 😅